src/codegen/sys/cargo_toml.rs

```diff
@@ -36,11 +36,6 @@ fn fill_empty(root: &mut Table, env: &Env, crate_name: &str) {
         let package = upsert_table(root, "package");
         set_string(package, "name", package_name);
         set_string(package, "version", "0.0.1");
-        set_string(
-            package,
-            "links",
-            nameutil::shared_libs_to_links(&env.namespaces.main().shared_libs),
-        );
         set_string(package, "edition", "2018");
```

Comment:
That would require to actually remove it here AFAIU

Reply:
Right, I was thinking that I'd just go through the crates that I care about and submit a PR manually…

Reply:
I'm not sure if removing things present in an existing Cargo file is a good idea though...

Reply:
OK, if both of you think that's OK then let's do that :)